code/__DEFINES/_globals.dm

```diff
@@ -58,3 +58,12 @@
 
 //Create an typed null global
 #define GLOBAL_DATUM(X, Typepath) GLOBAL_RAW(Typepath/##X); GLOBAL_UNMANAGED(X)
+
+GLOBAL_LIST_EMPTY(string_files)
```

Comment:
Defining a global in this file seems odd. Maybe this could go on `SSstrings` instead?

Reply:
I wasn't sure of the define order but yeah it should be there